Definitions/BlueMage.json

```diff
@@ -93,16 +98,22 @@
 ] },
 { "6B9" : "peculiar light", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5, "limittodamagetype": "Magic"} ] },
 { "6BA" : "diamondback", "potency": [ {"type": "DamageReceivedMultiplier", "amount": -90} ] },
-{ "846" : "harmonized", "potency": [ {"type": "damagedonemultiplier", "amount": 80, "limittodamagetype": "Physical"} ] },
-{ "847" : "cactguard", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5} ] },
-{ "849" : "physical attenuation", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5} ] },
+{ "846" : "harmonized", "potency": [ {"type": "damagedonemultiplier", "amount": 80, "limittodamagetype": "Physical", "limittoactioncategory": "spell"} ] },
+{ "847" : "cactguard", "potency": [ {"type": "DamageReceivedMultiplier", "amount": -5} ] },
+{ "84B" : "physical attenuation", "potency": [ {"type": "DamageReceivedMultiplier", "amount": 5, "limittodamagetype": "Physical"} ] },

```

Comment:
Condensed Libra: Physical Attenuation is 84B

849 is Astral Attenuation and 84A is Umbral Attenuation